RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
```

Comment:
this is not the right place to make these modifications.
XMLIdealGeometryESSource is present in some workflows already and this ``` geometryRP_cfi import * ``` overrides that.

@ianna 
please suggest a solution.
It's probably going to be something similar to what we do in 2017 for the hardcoded hcal conditions
It looks like there is interference with 2017 setup 
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14605/13382/runTheMatrix-results/10021.0_TenMuE_0_200+TenMuE_0_200_pythia8_2017_GenSimFull+DigiFull_2017+RecoFull_2017+HARVESTFull_2017/step3_TenMuE_0_200+TenMuE_0_200_pythia8_2017_GenSimFull+DigiFull_2017+RecoFull_2017+HARVESTFull_2017.log
```
   [1] Calling global beginRun for module EcalDeadCellTriggerPrimitiveFilter/'EcalDeadCellTriggerPrimitiveFilter'
   [2] Using EventSetup component CaloGeometryBuilder/'' to make data CaloGeometry/'' in record CaloGeometryRecord
   [3] Using EventSetup component HcalHardcodeGeometryEP/'' to make data CaloSubdetectorGeometry/'HCAL' in record HcalGeometryRecord
   [4] Using EventSetup component HcalDDDRecConstantsESModule/'hcalDDDRecConstants' to make data HcalDDDRecConstants/'' in record HcalRecNumberingRecord
   [5] Using EventSetup component HcalParametersESModule/'hcalParameters' to make data HcalParameters/'' in record HcalParametersRcd
Exception Message:
Not found OnlyForHcalSimNumbering but needed.

```